src/map/skill.cpp

```diff
@@ -16675,6 +16781,32 @@ int skill_unit_onplace_timer(struct skill_unit *unit, struct block_list *bl, t_t
 				skill_attack(skill_get_type(sg->skill_id), ss, &unit->bl, bl, sg->skill_id, sg->skill_lv, tick, flag);
 			}
 			break;
+
+		case UNT_TOTEM_OF_TUTELARY:
+			++sg->val1;
+			if( bl->type == BL_PC ) {
+				if (sg->val1 % 3 == 0) {
+					int hp = 500;
+
+					hp += 500 * sg->skill_lv;
+					hp += 50 * pc_checkskill( tsd, SOA_TALISMAN_MASTERY ) * sg->skill_lv;
+					hp += 50 * status_get_crt( ss ) * sg->skill_lv;
+					hp *= status_get_lv( ss ) / 100;
+
+					int sp = 0;
+
+					sp += 50 * sg->skill_lv;
+					sp += 5 * pc_checkskill( BL_CAST( BL_PC, ss ), SOA_TALISMAN_MASTERY ) * sg->skill_lv;
+					sp += 5 * tstatus->crt * sg->skill_lv;
+					sp *= 100 + status_get_lv( ss );
+					sp /= 150;
```

Comment:
```suggestion
					sp *= status_get_lv( ss ) / 100;
```
According to @datawulf